### DISCUSSION TITLE
Worried about project status

### DISCUSSION THREAD

**Discussion by rijenkii** [concourse/concourse#8864] [General]
Going to be blunt -- is this project currently abandoned/on pause?

- Only 3 people made commits in this project in 2023, totalling ~50 commits.
- Last update in the RFC repo was 2 years ago.

**Comment by marco-m-pix4d** [comment]
Duplicate of https://github.com/concourse/concourse/discussions/8782

**Comment by ardavanhashemzadeh** [comment]
Ditto, I opened a PR a few days back and asked a question about it in discussions a day later, I'd prefer not to reach out to individual devs, and I am willing to donate some time as a maintainer/moderator if its flesh and blood that's lacking
